### ISSUE TITLE
Shareable Profile Page

### ISSUE THREAD

**Issue by Apoorv-Saini** [stackup-dev/open-source-campaign#234]
**Name of Proposer**
Apsain

**Is your feature request related to a problem? Please describe.**
As we are learning new technologies, we are not able to show it to the world.

**Describe the solution you'd like**
A profile page which shows all our learnings and resume.

**Describe alternatives you've considered**
A career page where stackies can list their resume and learnings.

**Additional context**
It will help all stackies to get better job opportunities.

**Comment by Ri-Sharma** [comment]
Yes I think it would be a good idea, also we can also include other additional details like our github profile link, linkedin etc. It would also help other stackies to know each other.
And a search bar would be an great option where we could search other people profiles.
It can also include some other features like public or private, in which we could control what data to show and what data to hide from other people.
